<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>UML_KEditorClass</class>
 <widget class="QMainWindow" name="UML_KEditorClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>600</width>
    <height>400</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>UML_KEditorClass</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QGraphicsView" name="graphicsView">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>1200</width>
      <height>1200</height>
     </rect>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>600</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile_F">
    <property name="title">
     <string>File(&amp;F)</string>
    </property>
   </widget>
   <widget class="QMenu" name="menuEdit_E">
    <property name="title">
     <string>Edit(&amp;E)</string>
    </property>
   </widget>
   <addaction name="menuFile_F"/>
   <addaction name="menuEdit_E"/>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="actionUseCase"/>
   <addaction name="actionClass"/>
   <addaction name="separator"/>
   <addaction name="actionAssociation"/>
   <addaction name="actionComposition"/>
   <addaction name="actionGeneration"/>
   <addaction name="separator"/>
   <addaction name="actionText"/>
   <addaction name="actionChoose"/>
   <addaction name="actionGroup"/>
   <addaction name="actionUnGroup"/>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="actionUseCase">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="checked">
    <bool>false</bool>
   </property>
   <property name="icon">
    <iconset>
     <normaloff>Resources/image/usercase.png</normaloff>
     <normalon>Resources/image/usecase.png</normalon>Resources/image/usercase.png</iconset>
   </property>
   <property name="text">
    <string>UseCase</string>
   </property>
  </action>
  <action name="actionClass">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset>
     <normaloff>Resources/image/class.png</normaloff>Resources/image/class.png</iconset>
   </property>
   <property name="text">
    <string>Class</string>
   </property>
  </action>
  <action name="actionAssociation">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset>
     <normaloff>Resources/image/associate.png</normaloff>Resources/image/associate.png</iconset>
   </property>
   <property name="text">
    <string>Association</string>
   </property>
  </action>
  <action name="actionComposition">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset>
     <normalon>Resources/image/composition.png</normalon>
    </iconset>
   </property>
   <property name="text">
    <string>Composition</string>
   </property>
  </action>
  <action name="actionGeneration">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset>
     <normaloff>Resources/image/generation.png</normaloff>Resources/image/generation.png</iconset>
   </property>
   <property name="text">
    <string>Generation</string>
   </property>
  </action>
  <action name="actionChoose">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset>
     <normaloff>Resources/image/mouse.png</normaloff>Resources/image/mouse.png</iconset>
   </property>
   <property name="text">
    <string>Choose</string>
   </property>
  </action>
  <action name="actionText">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset>
     <normalon>Resources/image/Text.png</normalon>
    </iconset>
   </property>
   <property name="text">
    <string>Text</string>
   </property>
  </action>
  <action name="actionGroup">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset>
     <normaloff>Resources/image/group.png</normaloff>Resources/image/group.png</iconset>
   </property>
   <property name="text">
    <string>Group</string>
   </property>
  </action>
  <action name="actionUnGroup">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset>
     <normaloff>Resources/image/ungroup.png</normaloff>Resources/image/ungroup.png</iconset>
   </property>
   <property name="text">
    <string>UnGroup</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="uml_keditor.qrc"/>
 </resources>
 <connections/>
</ui>
